content/en/schemas/list.md

```diff
@@ -0,0 +1,13 @@
+---
+title: OpenTelemetry Schemas
+linkTitle: List
+type: docs
+body_class: ot-schema-list
+---
+
+The OpenTelemetry schemas hosted on this site are listed below. To learn more
+about schemas, see [Telemetry Schemas][].
+
+{{% schemas %}}
+
```

Comment:
```suggestion

> **Note**: This page is not intended to be consumed by automatic processes, as the format may change.
```

I think also this page should have such a disclaimer that this page is not here for being consumed by any kind of automatism.

Reply:
This list page is a normal web being served by the site. All pages are subject to change :). I'd vote against such a note, but I'll add one anyways and you can let me know what you think.